modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalImport/ModalImportContent.tsx

```diff
@@ -125,6 +125,7 @@ export function ModalImportContent({
 												fileName: '',
 												inputFile: null,
 											});
+											inputFileRef.current.value = '';
```

Comment:
You must clear this property in the `JSON.parse` catch as well, to avoid this behavior:



https://github.com/liferay-objects/liferay-portal/assets/61854510/8ce1b8d0-0e31-471a-8992-eb87eb15d7e0

Reply:
Done! :heavy_check_mark: